sympy/stats/crv_types.py

```diff
@@ -2525,6 +2527,86 @@ def Maxwell(name, a):
 
     return rv(name, MaxwellDistribution, (a, ))
 
+#-------------------------------------------------------------------------------
+# Moyal Distribution -----------------------------------------------------------
+class MoyalDistribution(SingleContinuousDistribution):
+    _argnames = ('mu', 'sigma')
+
+    @staticmethod
+    def check(mu, sigma):
+        _value_check(mu.is_real, "Location parameter must be real.")
+        _value_check(sigma.is_real and sigma > 0, "Scale parameter must be real\
+        and positive.")
+
+    def pdf(self, x):
+        mu, sigma = self.mu, self.sigma
+        num = exp(-(exp(-(x - mu)/sigma) + (x - mu)/(sigma))/2)
+        den = (sqrt(2*pi) * sigma)
+        return num/den
+
+    def _cdf(self, x):
+        mu, sigma = self.mu, self.sigma
+        return erfc(exp(-(x - mu)/(2*sigma))/sqrt(2))
```

Comment:
This can be removed.

Reply:
After removing `cdf` takes infinite time to evaluate.

Reply:
Removing only cdf doesnt affect the timings much. Therefore, only this can be removed and others have to be kept.

Reply:
I removed only `_cdf(self, x)` and checked but it still takes infinite time to evaluate.

Reply:
On my system it works fine,
```python
>>> from sympy.stats import Moyal, cdf
>>> mu = Symbol("mu", real=True)
>>> sigma = Symbol("sigma", positive=True, real=True)
>>> X = Moyal("X", mu, sigma)
>>> z = Symbol("z")
>>> cdf(X)(z)
   ⎛                  ⎛     μ   -z ⎞               ⎞     
   ⎜           -μ     ⎜    ───  ───⎟            -μ ⎟   μ 
   ⎜           ───    ⎜    2⋅σ  2⋅σ⎟            ───⎟  ───
   ⎜           2⋅σ    ⎜√2⋅ℯ   ⋅ℯ   ⎟            2⋅σ⎟  2⋅σ
√2⋅⎜- √2⋅√π⋅σ⋅ℯ   ⋅erf⎜────────────⎟ + √2⋅√π⋅σ⋅ℯ   ⎟⋅ℯ   
   ⎝                  ⎝     2      ⎠               ⎠     
─────────────────────────────────────────────────────────
                          2⋅√π⋅σ                         
>>> simplify(_)
       ⎛    μ - z⎞
       ⎜    ─────⎟
       ⎜     2⋅σ ⎟
       ⎜√2⋅ℯ     ⎟
1 - erf⎜─────────⎟
       ⎝    2    ⎠
```

Reply:
I just forgot to update my local environment. Yes, it worked. Thanks :)